Classes/ModList.lua

```diff
@@ -139,7 +139,6 @@ function ModListClass:ListInternal(context, result, cfg, flags, keywordFlags, so
 		for i = 1, #self do
 			local mod = self[i]
 			if mod.name == modName and mod.type == "LIST" and band(flags, mod.flags) == mod.flags and MatchKeywordFlags(keywordFlags, mod.keywordFlags) and (not source or mod.source:match("[^:]+") == source) then
-				local value
```

Comment:
Is this a fix related to culling? If not, please create new PRs for unrelated changes.

Reply:
It is just a cleanup of a useless variable I saw, not related to the change. I can revert it if you'd like.

Reply:
Please revert this change then.